taskvine/src/manager/vine_file.c

```diff
@@ -245,6 +245,14 @@ struct vine_file *vine_file_temp()
 	return vine_file_create("temp", 0, 0, 0, VINE_TEMP, 0, cache, 0);
 }
 
+struct vine_file *vine_file_pseudo_temp()
+{
+	// temp files are always cached at workers until explicitely removed.
+	vine_cache_level_t cache = VINE_CACHE_LEVEL_WORKFLOW;
+
+	return vine_file_create("temp", 0, 0, 0, VINE_FILE, 0, cache, 0);
```

Comment:
I'm not quite sure this one will work. Wouldn't all the temp files get the same name locally at the manager? Also I suggest changing the name to `vine_file_temp_no_peers` or something like that.